### DISCUSSION TITLE
Problema con las dependencias de Maven

### DISCUSSION THREAD

**Discussion by tobiascastaneda2000** [dds-jv/foro#347] [Consultas de herramientas] [answered]
Buenas noches!!

Me surgieron algunos inconvenientes con las configuración de intellij y las dependencias de Maven:

Primero, me pude instalar Intellij y Maven, sin embargo, al crear un proyecto nuevo con Maven, vi que en el archivo pom.xml me faltas dependencias por alguna razón. Estuve charlando con algunos compañeros y me faltan las dependencia de Junit asi como varios plugins que tengo entendido que tenia que venir por defecto. 
![2022-04-19](https://user-images.githubusercontent.com/62578744/164139524-9f58d948-1efd-40d6-b70a-c8746ecae3da.png)
Quería consultar de que pagina se debe copiar y pegar las dependencias, o hay algún paso que hice mal ? 

Disculpe las molestias. Saludos:

**Comment by gastonprieto** [comment, marked as answer]
Buenas @tobiascastaneda2000 

* Para macowins estas creando el proyecto de 0 ? el repositorio del assignment ya tiene el proyecto con todo lo que necesitas configurado, no solo pone junit sino que agrega un par de chequeos que son necesarios cumplir de estilo/formato + porcentaje de tests.

* Una duda que pusiste

> que tengo entendido que tenia que venir por defecto

Cuando vos creas un proyecto de 0, no te viene nada de junit u otro framework/libreria configurado, tenes que agregarlas vos. Fijate que en el apunte de maven explica lo que tenes que agregar (aunque como te comentaba, ahora no haria falta)

* Otra cosa que estoy viendo es que tiene configurado como compilador no java 8, que es el que se pide para el TP.

**Reply by gastonprieto** [reply]
te sirvio @tobiascastaneda2000 ?

**Reply by gastonprieto** [reply]
lo doy por cerrado @tobiascastaneda2000 , cualquier cosa volvenos a preguntar

**Reply by tobiascastaneda2000** [reply]
Buenas Tardes! Si me sirvió muchísimo y por suerte lo pude resolver, los tps como el anual generalmente al bajarse el repositorio ya vienen con las dependencias.

Además también me había instalado El Java 18, y para los repositorios de la materia estaban usando Java 8, por lo que tuve que me instale la otra versión, actualice los respectivos entornos de variable y cambie en la configuración de Intellij `Proyect Structure ` la version de Java. Luego de eso pude empezar a codear con tranquilidad y por ahora no me han surgido mas inconvenientes relacionados a las dependencias. (Solo el tp QMP lo habíamos creado en Java 18 antes de saber esto, por lo que al final para trabajar con ese tp en concreto cambiamos a esa versión)

Pido mil disculpas la tardanza y por no responder en su debido momento. 

Muchísimas Gracias
